index.js

```diff
@@ -26,22 +39,29 @@ Use getFinals to do the following:
 💡 HINT - you should be looking at the stage key inside of the objects
 */
 
-function getFinals(/* code here */) {
-    /* code here */
+function getFinals(array) {
+    const finalGames = array.filter(function(item){
+        return item.Stage === 'Final'
+    })
+    return finalGames
```

Comment:
Good job on this function. Since you're not changing the array after setting the variable finalGames you could just return array.filter instead of setting a variable only to return the value of the variable